.github/actions/pr-manager/index.js

```diff
@@ -0,0 +1,233 @@
+const core = require('@actions/core');
+const { context, getOctokit } = require('@actions/github');
+
+const token = core.getInput('github_token', { required: true });
+const octokit = getOctokit(token);
+const github = octokit.rest;
+
+// Regex that matches any snapshots, including -pre and -rc version numbers.
+const snapshotRegex = new RegExp("\d\dw\d\d[a-e]|\d\.\d\d\.\d\d?-(?:pre|rc)\d")
+
+// Any labels that are used in the code.
+const fcpLabel = 'final-comment-period';
+const tinyLabel = "s: tiny";
+const smallLabel = "s: small";
+const medLabel = "s: medium";
+const largeLabel = "s: large";
+const newLabel = "T: new";
+const refactorLabel = "T: refactor";
+const docLabel = "T: documentation";
+const toolLabel = "T: toolchain";
+
+var mappingsTeam;
+
+// Comment that gets posted when FCP has passed for a PR and it's good to merge.
+const fcpDoneComment = "FCP for this PR has passed! A Quilt Mappings team member should merge this.";
+
+// Get all members of Quilt's Mappings and Mappings Triage teams.
+async function getMappingsTeam() {
+    const mappingsTeamReq = await github.teams.listMembersInOrg({ org: "QuiltMC", team_slug: "mappings" });
+    const triageTeamReq = await github.teams.listMembersInOrg({ org: "QuiltMC", team_slug: "mappings-triage" });
+
+    // Merge both teams, removing duplicates
+    mappingsTeam = mappingsTeamReq.map((member) => member.login);
+    const triageTeam = triageTeamReq.map((member) => member.login);
+    triageTeam.forEach((member) => (mappingsTeam.some((existing) => predicate(member, existing)) ? null : mappingsTeam.push(member)));
+}
+
+// Check the size of a PR and add the appropriate label.
+async function addSizeLabel(size, num) {
+    if (size < 50) {
+        await github.issues.addLabels({ ...baseParameters, issue_number: num, labels: [tinyLabel] });
+    } else if (size < 200) {
+        await github.issues.addLabels({ ...baseParameters, issue_number: num, labels: [smallLabel] });
+    } else if (size < 700) {
+        await github.issues.addLabels({ ...baseParameters, issue_number: num, labels: [medLabel] });
+    } else {
+        await github.issues.addLabels({ ...baseParameters, issue_number: num, labels: [largeLabel] });
+    };
+}
+
+// Ensure the FCP label is on any PRs that are in FCP.
+// If it's there, no need to do anything.
+async function ensureFCPLabel(labels, num) {
+    // If FCP label doesn't exist, add it by calling the API.
+    if (!labels.includes(fcpLabel)) {
+        await github.issues.addLabels({ ...baseParameters, issue_number: num, labels: [fcpLabel] });
+    } else { };
```

Comment:
why this else?

Reply:
I missed that I had added that when doing my final code check, removed.